src/tripCard/TripCard.unit.tsx

```diff
@@ -21,22 +22,12 @@ function createPassengers(count: number): User[] {
 function createProps(props: Partial<TripCardProps> = {}): TripCardProps {
   return {
     href: '#',
-    itinerary: [
-      {
-        mainLabel: 'Paris',
-        subLabel: 'Porte de Vincennes',
-        time: '09:00',
-        isoDate: '2017-12-11T09:00',
-        distanceFromPoint: '1,5km',
-      },
-      {
-        mainLabel: 'Bordeaux',
-        subLabel: 'Gare Bordeaux Saint-Jean',
-        time: '12:00',
-        isoDate: '2017-12-11T12:00',
-        distanceFromPoint: '8km',
-      },
-    ],
+    itinerary: (
```

Comment:
Could you add some tests to check that the itinerary is displayed? I see no test related to it in this file.